ticket 2.2, 2.3 for review

YeabseraG opened the pull request:
2.2 header spacing adjusted, "edit profile" title/text added

2.3 profile picture made responsive based on mobile design (figma)

input fields created by somebody else, but seem to be correct and responsive

- overall responsiveness for the header seems to be incorrect but since it is not on the checklist for 2.2 i left it be

YeabseraG committed:
ticket 2.2, 2.3 for review

2.2 header spacing adjusted, "edit profile" title/text added

2.3 profile picture made responsive based on mobile design (figma)

input fields created by somebody else, but seem to be correct and responsive

- overall responsiveness for the header seems to be incorrect but since it is not on the checklist for 2.2 i left it be

GustavBoe commented on templates/students/edit-profile-student.html:
```diff
@@ -118,24 +118,29 @@
-			<a src="#" class="text-primary">
+			<a src="#" class="text-neutral-medium-gray font-Roboto text-base font-normal">
 				Profile</a>
+			<p>></p>
```
Should there be anything here?

GustavBoe replied:
Nevermind, I saw it now:) My bad.

GustavBoe reviewed (changes requested):
### For "Ticket 2.2":
 
- [x] Change the font size to rem to make it easier when working with responsiveness. 2rem should be the equivalent value.
 
### For "Ticket 2.3":
 
- [x] From screensize 640px-1023px I get both versions of both heading and placeholder image.
 
- [x] The fields need some more styling, the size could be converted to rem and the radius to rounded-lg.
 
_For desktop_
- width: 23.33919rem
- height: 3.2765rem
 
_For mobile_
- width: 19.5625rem
- height: 3.125 rem
 
- [x] The placeholder for the date-field needs to match Figma. Maybe this could be of any use: https://www.geeksforgeeks.org/how-to-set-placeholder-value-for-input-type-date-in-html-5/
 
 
**Looking forward to seeing the new version!**

YeabseraG committed:
update after review

YeabseraG committed:
Update edit-profile-student.html

GustavBoe commented on templates/students/edit-profile-student.html:
```diff
@@ -118,61 +118,66 @@
-			<section class="">
-				<form action="#" class="w-full lg:px-40">
+			<section class="mt-10 mb-[4.35rem] mx-4 md:p-10 lg:border-3 h-[71.33463rem] border-neutral-medium-gray">
+				<form action="#" class="w-[23.33919px] h-[3.2765rem] lg:px-40">
```
**w-[23.33919px]** should be **w-[23.33919rem]**
I think that **lg:px-40** messes with the length of your fields. Please try another solution, preferably using rem.

Also, I get two borders. You can delete the border around the table. I think that you can find it on line 146.

YeabseraG committed:
update ticket 2.2, 2.3

px changed to rem
lg-px removed, input fields no longer change with size.
2nd border deleted

YeabseraG committed:
Final Update (hopefully)

style changes for input fields

GustavBoe committed:
Merge branch 'develop' into epic-4--ticket-2.2,-2.3,-2.5

GustavBoe reviewed (approved):
Lovely work Yeabsera! The buttons looks fantastic. Will approve and merge now.

GustavBoe reviewed (approved):
Great work Yeabsera, the buttons look fantastic! Will approve and merge.
